patches/0009-initialize-setproctitle-where-needed.patch

```diff
@@ -48,5 +48,39 @@ index c7935bf..44caa80 100644
  		switch (ch) {
  		case 'd':
 -- 
-2.4.3.573.g4eafbef
+2.0.4
+
+From 8b01bffe9f2def35b180118f9f41e47c4fc0a9cc Mon Sep 17 00:00:00 2001
+From: "Paul B. Henson" <henson@acm.org>
+Date: Sun, 5 Jul 2015 14:06:36 -0700
+Subject: [PATCH 09/12 part 2] Move __progname initialization to before first use.
+
+---
+ src/usr.sbin/ntpd/ntpd.c | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/src/usr.sbin/ntpd/ntpd.c b/src/usr.sbin/ntpd/ntpd.c
+index 44caa80..310e808 100644
+--- a/src/usr.sbin/ntpd/ntpd.c
++++ b/src/usr.sbin/ntpd/ntpd.c
+@@ -131,6 +131,8 @@ main(int argc, char *argv[])
+ 	struct passwd		*pw;
+ 	extern char		*__progname;
+ 
++	__progname = get_progname(argv[0]);
```

Comment:
I have a patches branch that these are derived from here: https://github.com/busterb/openntpd-openbsd/tree/patches

That makes it easier to rebase the patches as needed. I'll just squash your fixes into that and regen the patch files. Thanks!